template/src/ontology/Makefile.jinja2

```diff
@@ -535,13 +535,17 @@ IMP_LARGE=true # Global parameter to bypass handling of large imports
 
 {% for ont in project.import_group.products %}
 
+{% if ont.mirror_type != 'none' -%}
 ## ONTOLOGY: {{ ont.id }}
+{% endif -%}
```

Comment:
Is this "endif" at the right place? Why you you skop the ONTOLOGY comment but not the "description" comment?

Reply:
I thought again, and it's better if the comments stay there with the comment below when there isn't a mirror.
I skipped only the phony/precious part.